prow/config/jobs/operator.yaml

```
@@ -24,6 +24,11 @@ jobs:
     command: [make, gen-check]
     modifiers: [optional]
 
+  - name: e2e
+    command: [entrypoint, make, e2e]
+    requirements: [kind]
+    modifiers: [optional]
+
```

```suggestion
    repos: [istio/istio]

```

to automatically checkout istio. Can be a followup